src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +65,9 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Lets us know whether we have to write to file
```

Comment:
This method should have a detailed comment regarding what the method aims to do and why.